From 86564ef26341020316cd8a27c704ef1dc5a6129b Mon Sep 17 00:00:00 2001
From: Jon Dufresne <jon.dufresne@gmail.com>
Date: Thu, 18 Aug 2016 17:17:24 -0700
Subject: [PATCH] Remove LOCALE from str regular expression

Fixes warning:

DeprecationWarning: LOCALE flag with a str pattern is deprecated. Will be an error in 3.6

Fixes #90
---
 xlwt/ExcelFormulaLexer.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xlwt/ExcelFormulaLexer.py b/xlwt/ExcelFormulaLexer.py
index 46a2bff..8ff7386 100644
--- a/xlwt/ExcelFormulaLexer.py
+++ b/xlwt/ExcelFormulaLexer.py
@@ -3,7 +3,7 @@
 
 from .antlr import EOF, CommonToken as Tok, TokenStream, TokenStreamException
 from . import ExcelFormulaParser
-from re import compile as recompile, LOCALE, IGNORECASE, VERBOSE
+from re import compile as recompile, IGNORECASE, VERBOSE
 
 
 int_const_pattern = r"\d+\b"
@@ -50,7 +50,7 @@
 
 _re = recompile(
     '(' + ')|('.join([i[0] for i in pattern_type_tuples]) + ')',
-    VERBOSE+LOCALE+IGNORECASE)
+    VERBOSE+IGNORECASE)
 
 _toktype = [None] + [i[1] for i in pattern_type_tuples]
 # need dummy at start because re.MatchObject.lastindex counts from 1
